oshmem/mca/spml/ucx/spml_ucx.h

```diff
@@ -68,6 +68,7 @@ struct mca_spml_ucx {
 
     int                      priority; /* component priority */
     bool                     enabled;
+    spml_ucx_mkey_t          *(*mca_spml_ucx_get_mkey_slow)(int pe, void *va, void **rva);
```

Comment:
this could be renamed to just `get_mkey_slow`

Reply:
done